### PR TITLE
Adding property to expose the SNICert property

### PR DESCRIPTION
**Pull request by hsmade**

### CODE REVIEW

**Comment by giorgos-nikolopoulos** [comment]
So the patch is in the right direction but it has some problems.

* The key name should be hinting at the type `sslsnicertkeys` would be an example
* The type should be `TypeSet`. It is an error to bind the same certificate name multiple times.
* There is no handling for the update of the resource. Someone might add/remove sni certificates to an existing lb vserver.
* It handles the default certificate defined under the key `sslcertkey` the same as sni certificates. The default certificate is what is used when the sni lookup fails to find a relevant certificate. So it should be bound to lb vserver without the `snicert` option set to `true`

@hsmade 
Let us know if you are willing to have a go at it, otherwise we will take up the task to add  the sni certificates bindings.

File: netscaler/resource_lbvserver.go [hunk 1]
```diff
@@ -529,6 +534,18 @@ func createLbvserverFunc(d *schema.ResourceData, meta interface{}) error {
 			return fmt.Errorf("[ERROR] netscaler-provider: Specified ssl cert key does not exist on netscaler!")
 		}
 	}
+	sslcertkeys := []interface{}{sslcertkey}
```
Comment (giorgos-nikolopoulos) [inline netscaler/resource_lbvserver.go]: The default certificate should not be added to the list of SNI certificates.

**Comment by hsmade** [comment]
This part of the PR was wrong, indeed. I have one that is a bit more up to date, but throws a panic while writing.

I've abandoned the idea of trying to add certs in the project that I was using this for. So I'll spend my time on other stuff. Feel free to take it up.

**Comment by giorgos-nikolopoulos** [comment]
This PR feature request is implemented in #45

**Pull request closed without merging**